packages/datadog-instrumentations/src/mongoose.js

```diff
@@ -128,22 +128,21 @@ addHook({
                     const resolve = arguments[0]
                     const reject = arguments[1]
 
-                    // not using shimmer here because resolve/reject could be empty
-                    arguments[0] = function wrappedResolve () {
```

Comment:
it says it doesn't use shimmer for a specific reason here, is that handled correctly by shimmer now ?